### ISSUE TITLE
messageString is missing the `omitempty` tag

### ISSUE THREAD

**Issue by another-rex** [owenrumney/go-sarif#67]
`messageStrings` added in #63 is missing the `omitempty` tag so it's always showing up in results, e.g.:

```
"rules": [
  {
    "id": "CVE-123-456",
    "shortDescription": {
      "text": "..."
    },
    "messageStrings": null
  }
]
````

**Comment by owenrumney** [comment]
resolved in v0.0.2

**Comment by another-rex** [comment]
Thank you!
